version.bzl

```diff
@@ -12,10 +12,8 @@ the app (that are potentially not broadly released yet).
 MAJOR_VERSION = 0
 MINOR_VERSION = 13
 
-# TODO(#4419): Remove the Kenya-specific alpha version code.
 OPPIA_DEV_VERSION_CODE = 166
```

Comment:
Let's go ahead and bump these versions right away so that there isn't a gap (that means setting ``OPPIA_GA_VERSION_CODE`` to 167 and then adding one for each version above it).

Reply:
Just double confirm you mean OPPIA_GA_VERSION_CODE to 161 or 167? Here are the current codes I have from latest develop branch
"OPPIA_DEV_VERSION_CODE = 166
OPPIA_DEV_KITKAT_VERSION_CODE = 165
OPPIA_ALPHA_VERSION_CODE = 163
OPPIA_ALPHA_KITKAT_VERSION_CODE = 162
OPPIA_BETA_VERSION_CODE = 161
OPPIA_GA_VERSION_CODE = 160"

Reply:
Version code updated

Reply:
The latest looks correct--thanks!